pkg/logs/internal/tailers/file/tailer.go

```diff
@@ -151,6 +154,12 @@ func NewTailer(opts *TailerOptions) *Tailer {
 	fileRotated := opts.Rotated
 	opts.Info.Register(bytesRead)
 
+	timeWindow := 24 * time.Hour
+	totalBucket := 24
+	bucketSize := timeWindow / time.Duration(totalBucket)
+	movingSum := util.NewMovingSum(timeWindow, bucketSize, getCurrentTime())
```

Comment:
```suggestion
	movingSum := util.NewMovingSum(timeWindow, bucketSize, clock.New())
```
You can just use `clock.New()` here and get rid of `getCurrentTime`